projects/vdk-control-cli/tests/resources/test-job-python-version/config.ini

```diff
@@ -0,0 +1,10 @@
+[owner]
+team = test-team
+[job]
+schedule_cron = 16 11 * * *
+python_version = 3.8
```

Comment:
Are there any other use cases known? For example, what's the expectation in case of a typo / non-configured valid version ?

Reply:
If the python version set in the config.ini is not part of the supported python versions, the Control Service returns 403 Bad Request, with a message indicating what python version was passed, and what python versions are supported.

Reply:
@doks5 is this scenario test-covered?

Reply:
We have tests covering python_version mismatches in the Control Service test suit. I don't think it would be useful to have such tests in the cli, as it does not really validate anything, it's just a medium that passes data to the Control Service.